.gitignore

```
@@ -92,6 +92,7 @@ cpu.out
 /VERSION
 /.air
 /.go-licenses
+/vscode/*
```

```suggestion
/.vscode
```

Pretty sure it is with the `.`

---

yup, you're right. thanks for catching :)